src/main/iidxhook9/fs-hook.c

```diff
@@ -0,0 +1,50 @@
+#define LOG_MODULE "fs-hook"
+
+#include <stdint.h>
+#include <string.h>
+
+#include "hook/table.h"
+
+#include "imports/avs.h"
+
+#include "iidxhook9/fs-hook.h"
+
+#include "util/log.h"
+#include "util/str.h"
+
+static void *(*real_avs_fs_mount)(const char *dest, const char *src, const char *fs_type, const char *options);
+static void *my_avs_fs_mount(const char *dest, const char *src, const char *fs_type, const char *options);
+
+static const struct hook_symbol avs_fs_hook_syms[] = {
+    {.name = "XCgsqzn000004b", // avs_fs_mount
+     .ordinal = 76,
+     .patch = my_avs_fs_mount,
+     .link = (void **) &real_avs_fs_mount},
+};
+
+static void *my_avs_fs_mount(const char *dest, const char *src, const char *fs_type, const char *options)
+{
+    // quick check for "F:\"
+    if (src[0] == 'F' && src[1] == ':' && src[2] == '\0') {
```

Comment:
question: Is it always capital 'F'? Seen some odd stuff in the past where they also used the lower-case drive letters because reasons.

Reply:
Yes